tools/single_use/Normalise_Consent_Data.php

```diff
@@ -307,8 +307,7 @@
     $columnName = $column['Column_name'];
     $output .= "ALTER TABLE participant_status DROP COLUMN " . $columnName . ";\n";
 }
-$filename = __DIR__ . "/../../SQL/Cleanup_patches/delete_old_consent_tables.sql";
+$filename = __DIR__ . "/../../SQL/Archive/20.0/cleanup/delete_old_consent_tables.sql";
```

Comment:
`/SQL/Archive/20.0/cleanup` directory doesn't exist yet so this throws an error. should we create the cleanup directory as part of this PR?

Reply:
the same change should also be made to line 7 in the starting comments

Reply:
yes